srcpkgs/rdma-core/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="pkgconf"
+makedepends="libnl3-devel eudev-libudev-devel python3-devel valgrind-devel"
+short_desc="RDMA core userspace libraries and daemons"
+maintainer="Rich G <rich@richgannon.net>"
+license="GPL-2.0-or-later, Linux-OpenIB, BSD-2-Clause-FreeBSD"
```

Comment:
Please pick between GPL-2.0-or-later or Linux-OpenIB (It is dual licensed and you're the maintainer)

See https://github.com/linux-rdma/rdma-core/blob/master/COPYING.md for the licensing situation

Reply:
D'oh, that makes sense.